addons/osio-addon/src/main/java/io/fabric8/launcher/osio/steps/OpenShiftSteps.java

```diff
@@ -115,15 +115,15 @@ private BuildConfig createBuildConfigObject(OsioProjectile projectile, GitReposi
         String gitUrl = repository.getGitCloneUri().toString();
         BuildConfig buildConfig = Builds.createDefaultBuildConfig(projectile.getOpenShiftProjectName(), gitUrl, getJenkinsUrl());
         Map<String, String> currentAnnotations = KubernetesHelper.getOrCreateAnnotations(buildConfig);
-        currentAnnotations.putAll(getBuildConfigAnnotations(projectile));
+        currentAnnotations.putAll(getBuildConfigAnnotations(projectile, repository.getFullName()));
         return buildConfig;
     }
 
-    private Map<String, String> getBuildConfigAnnotations(OsioProjectile projectile) {
+    private Map<String, String> getBuildConfigAnnotations(OsioProjectile projectile, String gitRepositoryName) {
```

Comment:
It would be better if the `GitRepository` object was used in this parameter. Typesafe ftw! ;)

Reply:
really if you only need one thing from that object why pass in the entire object

Reply:
Typesafe. A String doesn't have any meaning. A GitRepository does

Reply:
Why are you worried about passing the object? It's not a remote call